docs/source/techniques/SANS-Toml-v1.rst

```diff
@@ -1152,128 +1820,247 @@ in :ref:`algm-TOFSANSResolutionByPixel`.
 **Existing Example:**
 
 ..  code-block:: none
 
-    QRESOL/MODERATOR=moderator_rkh_file.txt
+    QRESOL/A1=30
 
-**Existing Replacement**
+**Replacement Example**
 
 ..  code-block:: none
 
   [q_resolution]
-    moderator_file = moderator_rkh_file.txt
+    source_aperture = 0.03
+
+QRESOL/A2=x
+-----------
 
+This command was used to specify the **sample** aperture *diameter* to be
+used in the estimation of the Q-resolution. Also see QRESOL/A1=x and
+QRESOL/H2=x.
 
-QRESOL[/ON][/OFF]
------------------
+For more information, see the
+`TOFSANSResolutionByPixel <https://docs.mantidproject.org/nightly/algorithms/TOFSANSResolutionByPixel-v1.html>`_
+algorithm description.
+
+**This command assumes that the data were collected on an instrument with
+pinhole collimation!**
+
+The sample aperture will normally be smaller than the source aperture!
+
+Note that because the source aperture size is frequently altered, the ISIS
+SANS Group decided to place the TOML replacement in the [instrument.configuration]
+block at the top of TOML User Files instead of the [q_resolution] block.
 
 **Replacement**
 
 ..  code-block:: none
 
-  [q_resolution]
-    enabled = true  # Or false
+  [instrument.configuration]
+    sample_aperture_diameter = x
 
 **Existing Example:**
 
 ..  code-block:: none
 
-    QRESOL/ON
+    QRESOL/A2=20
 
-**Existing Replacement**
+**Replacement Example**
 
 ..  code-block:: none
 
-  [q_resolution]
-    enabled = true  # Or false
-
+  [instrument.configuration]
+    sample_aperture_diameter = 0.02
 
-QRESOL[/H1=x][/H2=wx][/W1=x][/W2=x]
+QRESOL[/H1=x1][/W1=y1][/H2=x2][/W2=y2]
 -----------------------------------
 
+This command was used to specify the **source and sample** slit sizes to be
+used in the estimation of the Q-resolution. Also see QRESOL/A1=x and
+QRESOL/A2=x.
+
+For more information, see the
+`TOFSANSResolutionByPixel <https://docs.mantidproject.org/nightly/algorithms/TOFSANSResolutionByPixel-v1.html>`_
+algorithm description.
+
+**This command assumes that the data were collected on an instrument with
+slit/jaw collimation!**
+
+The sample slit size will normally be smaller than the source slit size! But
+the heights and widths of a slit do not have to be the same.
+
 **Replacement**
 
 ..  code-block:: none
 
   [q_resolution]
-    h1 = x
-    h2 = x
-    w1 = x
-    w2 = x
+    h1 = x1
+    w1 = y1
+    h2 = x2
+    w2 = y2
 
 **Existing Example:**
 
 ..  code-block:: none
 
     QRESOL/H1=16.0
-    QRESOL/H2=8.0
     QRESOL/W1=16.0
+    QRESOL/H2=8.0
     QRESOL/W2=8.0
 
-**Existing Replacement**
+**Replacement Example**
 
 ..  code-block:: none
 
   [q_resolution]
-    h1 = 16.0
-    h2 = 8.0
-    w1 = 16.0
-    w2 = 8.0
+    h1 = 0.016
+    w1 = 0.016
+    h2 = 0.008
+    w2 = 0.008
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+.. David you had these as mm even in TOML; I assume that was a mistake?
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
```

Comment:
Yeah, everything is in meters so that mistakes like this becomes obvious. Thanks for fixing thank 👍